src/index.js

```diff
@@ -0,0 +1,265 @@
+import { setDate } from './modules/helpers';
+// eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files

Reply:
Actualy, this is what i meant by  disabling the linters. I didn't changed the files.